Resources/Prototypes/Entities/Objects/Materials/materials.yml

```diff
@@ -534,3 +534,45 @@
     count: 1
   - type: Item
     size: 1
+
+- type: entity
+  parent: MaterialBase
+  id: MaterialBones
+  name: bones
+  description: Apply to a wolf
```

Comment:
Not really sure what this means?

Reply:
its just a bones material with a minecraft easter egg. I have not find anything better than that...